### PR TITLE
Fronted #2

### PR DESCRIPTION
**Pull request by emqi**
I double dare you motherlover, say something about hardcoded data one more time...

### CODE REVIEW

File: half-lajf-app/src/components/layout/Sidenav.js [hunk 1]
```diff
@@ -1,27 +1,44 @@
...
+          <li>
+            <ul className="collapsible">
+              <li>
+                <div className="collapsible-header black-text"><i className="material-icons collapsible-secondary">arrow_drop_down</i><b>Categories</b></div>
+                <div className="collapsible-body black-text">
+                  <a href='#' className="black-text sidenav-close"><span style={{marginLeft: 16}}>Category</span></a>
+                  <a href='#' className="black-text sidenav-close"><span style={{marginLeft: 16}}>Category</span></a>
+                  <a href='#' className="black-text sidenav-close"><span style={{marginLeft: 16}}>Category</span></a>
+                  <a href='#' className="black-text sidenav-close"><span style={{marginLeft: 16}}>Category</span></a>
+                  <a href='#' className="black-text sidenav-close"><span style={{marginLeft: 16}}>Category</span></a>
```
Comment (osiewicz) [inline half-lajf-app/src/components/layout/Sidenav.js:30]: Hardcoded value. What about adding new categories in the future?

File: half-lajf-app/src/components/memes/Comment.js [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+            <p>
+              <span className="commentAuthor">
+                <b>Twoja stara</b>
+              </span><br/>
+              <span className="commentDate" style={{fontSize: 11}}>10:34:01 PM 25/05/2019</span>
+            </p>
+          </div>
+          <div className="col s9">
+            <p className="commentText">
+              To jest testowy komentarz. Wjebałem tu w kurwe długą linię tekstu, żeby zobaczyć co się będzie działo. Lubię memy i wojaki. Elo elo elo elo elo elo tego typu benc. BONUS BONUS BONUS BONUS BONUS
```
Comment (osiewicz) [inline half-lajf-app/src/components/memes/Comment.js:18]: Hardcoded value.

File: half-lajf-app/src/components/memes/Comment.js [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
+  render() {
+    return (
+      <div className="comment">
+        <div className="row">
+          <div class="col s3">
+            <p>
+              <span className="commentAuthor">
+                <b>Twoja stara</b>
+              </span><br/>
+              <span className="commentDate" style={{fontSize: 11}}>10:34:01 PM 25/05/2019</span>
```
Comment (osiewicz) [inline half-lajf-app/src/components/memes/Comment.js:13]: Hardcoded value.

File: half-lajf-app/src/components/memes/Comment.js [hunk 4]
```diff
@@ -1,11 +1,27 @@
...
-      <p>To jest testowy komentarz. Wjebałem tu w kurwe długą linię tekstu, żeby zobaczyć co się będzie działo. Lubię memy i wojaki. Elo elo elo elo elo elo tego typu benc.</p>
+export class Comment extends Component {
+  render() {
+    return (
+      <div className="comment">
+        <div className="row">
+          <div class="col s3">
+            <p>
+              <span className="commentAuthor">
+                <b>Twoja stara</b>
```
Comment (osiewicz) [inline half-lajf-app/src/components/memes/Comment.js:11]: Hardcoded value.

File: half-lajf-app/src/components/memes/Counter.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import './Counter.css'
+
+const Counter = () => {
+  return (
+    <div className="counter center">
+      <p>
+        <i className="material-icons icon-red">thumb_down</i>
+          <span className="like-counter">Shitload of likes.</span>
```
Comment (osiewicz) [inline half-lajf-app/src/components/memes/Counter.js:9]: Hardcoded value.

File: half-lajf-app/src/components/memes/MemeDetails.js [hunk 6]
```diff
@@ -1,25 +1,61 @@
...
     return (
       <div className="container section meme-details">
         <div className="row">
-          <div className="col s12 m6 offset-m3">
+          <div className="col s12 m8 offset-m2">
             <div className="card z-depth-1">
               <div className="card-content grey-text text-darken-3">
                 <span style={{fontWeight: "bold"}} className="card-title">Meme title</span>
-                <p>Posted by TheWojak2</p>
+                <p>Posted by <b>TheWojak2</b>, 1 day ago</p>
```
Comment (osiewicz) [inline half-lajf-app/src/components/memes/MemeDetails.js:37]: Hardcoded value.

File: half-lajf-app/src/components/memes/MemeDetails.js [hunk 7]
```diff
@@ -1,25 +1,61 @@
...
+  }
   render() {
     return (
       <div className="container section meme-details">
         <div className="row">
-          <div className="col s12 m6 offset-m3">
+          <div className="col s12 m8 offset-m2">
             <div className="card z-depth-1">
               <div className="card-content grey-text text-darken-3">
                 <span style={{fontWeight: "bold"}} className="card-title">Meme title</span>
```
Comment (osiewicz) [inline half-lajf-app/src/components/memes/MemeDetails.js:36]: Hardcoded value.

File: half-lajf-app/src/components/user/Profile.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+          <p>Full name: </p>
+          <p>Email: </p>
+          <p>Password </p>
+          <p>Link your social media accounts:</p>
+          <p><a href="#">
+            <img src={require('../../resources/iconfinderGoogle.png')} alt="Google login" style={{width: 24, height: 24}}/>
+          </a> Currently unlinked</p>
+          <p><a href="#">
+            <img src={require('../../resources/iconfinderFacebook.png')} alt="Google login" style={{width: 24, height: 24}}/>
+          </a> Currently unlinked</p>
```
Comment (osiewicz) [inline half-lajf-app/src/components/user/Profile.js:20]: Hardcoded value, but I guess that's fine.

**Review by osiewicz — APPROVED** [review]

File: half-lajf-app/src/components/layout/Sidenav.js (hunk 1 above)
Comment (emqi) [reply half-lajf-app/src/components/layout/Sidenav.js]: Don't worry, I'm gonna fix it soon by commiting my fist to your face :)

**Pull request merged**